frontstage/controllers/case_controller.py

```diff
@@ -197,19 +145,19 @@ def get_cases_by_party_id(party_id, case_events=False):
     return response.json()
 
 
-def get_eq_url(case_id, party_id):
+def get_eq_url(case_id, party_id, business_party_id, survey_short_name):
     logger.debug('Attempting to generate EQ URL', case_id=case_id, party_id=party_id)
 
     case = get_case_by_case_id(case_id)
 
-    if case['caseGroup']['caseGroupStatus'] == 'COMPLETE':
+    if case['caseGroup']['caseGroupStatus'] == 'COMPLETE' or case['caseGroup']['caseGroupStatus'] == 'COMPLETEDBYPHONE':
```

Comment:
`if case['caseGroup']['caseGroupStatus'] in ('COMPLETE', 'COMPLETEDBYPHONE'):`
saves a bit of code